perses/app/relative_point_mutation_setup.py

```diff
@@ -139,8 +139,8 @@ def __init__(self,
                  conduct_endstate_validation=True,
                  flatten_torsions=False,
                  flatten_exceptions=False,
-                 rest_radius=0.3,
-                 w_scale=0.3,
+                 rest_radius=0.3 * unit.nanometer,
+                 w_lifting=0.3 * unit.nanometer,
```

Comment:
you found that 0.3nm is a good lifting value?

Reply:
Yes, from preliminary experiments I found that 0.3 was good. I still need to run the `w_lifting` optimization experiments for the paper, but hopefully the results will be similar to the preliminary experiments.